src/utils/strings.js

```diff
@@ -5,6 +5,7 @@ const strings = {
     leastOneField: "At least one field must be filled out.",
     unknownError: "An unknown error occured.",
     required: "This field is required.",
+    emailsDontmatch: "Your emails do not match.",
```

Comment:
Please change to "Emails must match" or similar. 

"Your" is not needed. These error messages are written as what should be done, as opposed to what the user has done incorrectly. More positive.

Reply:
Don't we want to keep the original Gravity Forms error messages? I just copied them from the plugin